### ISSUE TITLE
Cancellation of RedDotRubyConf 2020

### ISSUE THREAD

**Issue by winston** [reddotrubyconf/rdrc-2020#1]
# tl;dr

RedDotRubyConf 2020 originally scheduled on September 9 and 10 will be cancelled.
We will restart planning for the conference when DORSCON in Singapore is deactivated.

# The Why

Due to the Covid-19 situation, we, have decided to cancel RedDotRubyConf originally scheduled on September 9 and 10, 2020.

This decision was made after the assessing the risks to our attendees, speakers and volunteers should RedDotRubyConf proceed as planned.

Singapore is currently operating with the [Disease Outbreak Response System Condition](https://www.gov.sg/article/what-do-the-different-dorscon-levels-mean) (DORSCON) framework and the [current level is Orange](https://www.moh.gov.sg/news-highlights/details/risk-assessment-raised-to-dorscon-orange). 

At the Orange level, the recommendation from the Government is to cancel or defer non-essential large-scale events. 

For those who choose to proceed, they should take all necessary precautions. These include:
- Carry out temperature screening;
- Look out for respiratory symptoms such as cough or runny nose, and deny entry to unwell individuals;
- Remind participants not to attend if the participants have recent travel history to mainland China, and require travel declaration, if possible;
- Ensure that event venues are ventilated and are adequately equipped with facilities for hand washing; and
- Increase the frequency of cleaning commonly used areas; and
- Maintain a registration list of participants, if practical.

As a result of this, many events in Singapore that are happening within the next 3 months have been postponed or cancelled.

We have 2 options to work with:
1. Continue our planning for RedDotRubyConf on September 9 and 10, 2020
2. Cancel RedDotRubyConf

To go with Option (1), we can consider the 2 scenarios:
- a) The Covid-19 virus is cleared, and the DORSCON framework is deactivated.
- b) The Covid-19 virus is still present, and the DORSCON framework is still active.

Given the fluid situation around Covid-19 and looking at current data trends, 
it's untenable and irresponsible for us to even consider Scenario (a).

The future is hard to predict and it's only sensible for us to recognise the current DORSON level 
we are in and assume the worse - which means Scenario (b).

To plan a conference around Scenario (b) though has a lot of inherent risks, and associated costs to attempt to mitigate the risks. Questions we might need to consider include:
- Will attendees want to travel to Singapore?
- Will attendees have problem entering Singapore?
- How do we ensure that we are all safe at the conference? 
  - Do we need temperature screening procedures?
  - Do we need a bigger than usual event space that's well-ventilated?
- How would we do contact tracing if the need arises?

.. and lots more.

In light of that, we have hence decided on **Option (2) - Cancel RedDotRubyConf**.

We are volunteers, with a love for Ruby, and we want the conference to be an event where everyone feels safe and enjoys. 

Given that we are still early in our planning stage, then it's responsible for us to take a pause now after evaluation of the situation, and before we commit to more costs.

We are truly saddened to announce this, but we will restart planning for the conference when the dust has settled. 

Thank you! ❤️
